workflow-management/templates/deployment.yaml

```diff
@@ -58,15 +58,15 @@ spec:
             - name: NEXTFLOW_K8S_RUNASUSER
               value: {{ .Values.appConfig.nextflow_k8s_runAsUser | quote }}
             - name: NEXTFLOW_K8S_SERVICEACCOUNT
-              value: {{ .Values.appConfig.nextflow_k8s_serviceAccount | quote }}
+              value: {{ include "workflow-management.serviceAccountName" . }}
```

Comment:
can/should this be not hardcoded, ie. `workflow-management.`

Reply:
this can be defined by the user, will change